src/infrastructure/figma/transformers/figma-node-transformer.ts

```diff
@@ -72,6 +81,7 @@ export const tryTransformNodeToAtlassianDesign = ({
 			: AtlassianDesignStatus.NONE,
 		type: mapNodeTypeToDesignType(node.type),
 		lastUpdated: extra.lastModified,
+		lastUpdatedBy: fileMetaResponse.file.last_touched_by,
```

Comment:
**issue:** Please, do a manual mapping here as well to include only necessary field and exclude unnecessary (e.g., as `email`).

Reply:
Sorry missed that 🤦